ledger/sandbox/BUILD.bazel

```diff
@@ -250,6 +252,32 @@ client_server_test(
     ],
 ) if not is_windows else None
 
+client_server_test(
```

Comment:
Please add a similar conformance test for H2 to `/ledger/api-server-damlonx/BUILD.bazel` with the target `//ledger/api-server-damlonx:reference-v2` instead of the sandbox.

Reply:
Adding this with https://github.com/digital-asset/daml/pull/2698 - somehow I missed this when I merged.